### ISSUE TITLE
[dev/data-plane] Some help messages are not wrapped 

### ISSUE THREAD

**Issue by agileshaw** [canonical/charmed-openstack-upgrader#303]
The help messages of `hypervisor` sub-command and `--verbose` option are not properly wrapped, which hurts the readability.
```
Usage: cou plan hypervisors [options]

Show the steps for upgrading nova-compute machines.
This is possible only if control-plane has been fully upgraded,
otherwise an error will be thrown.
Note that only principal applications colocated with nova-compute units that support action-managed upgrades are within the scope of this command. Other principal applications (e.g. ceph-osd) and subordinates can be upgraded via the data-plane subcommand.

Options:
  -h, --help            Show this help message and exit.
  --model MODEL_NAME    Set the model to operate on.
                        If not set, the currently active Juju model will be used.
  --backup, --no-backup
                        Include database backup step before cloud upgrade.
                        Default to enabling database backup.
  --force               Force the plan/upgrade of non-empty hypervisors.
  --verbose, -v         Increase logging verbosity in STDOUT. Multiple 'v's yield progressively more detail (up to 4).
                        Note that by default the logfile will include standard logs from juju and websockets, as well as debug logs from all other modules. To also include the debug level logs from juju and websockets modules, use the maximum verbosity.
  --quiet, -q           Disable output in STDOUT.
  --machine MACHINES, -m MACHINES
                        Specify machine id(s) to upgrade.
                        This option accepts a single machine id as well as a stringified comma-separated list of ids,
                        and can be repeated multiple times.
  --availability-zone AVAILABILITY_ZONES, --az AVAILABILITY_ZONES
                        Specify availability zone(s) to upgrade.
                        This option accepts a single availability zone as well as a stringified comma-separated list of AZs,
                        and can be repeated multiple times.
```

**Comment by rgildein** [comment]
@agileshaw this should be closed or not?

**Comment by agileshaw** [comment]
Yes, the changes addressing this issue has been merged to `main` as part of #328 .
